### ISSUE TITLE
Provide a SvelteKit template

### ISSUE THREAD

**Issue by vhscom** [framework7io/framework7#3984]
**Is your feature request related to a problem? Please describe.**
I wanted to start an app using SvelteKit but I can't find any documentation.

**Describe the solution you'd like**
A template or tracking issue to provide SvelteKit support.

**Describe alternatives you've considered**
I looked at the SvelteKit repo but there's only one search result for Framework7 and it looks like someone may have gotten it working using some Vite customizations.

**Additional context**
As much as possible I'd prefer to use SvelteKit for SSR and routing. I understand Framework7 may be shipping its own router but does that mean it cannot be swapped out for something else?

**Comment by dariosalvi78** [comment]
someone managed to put together a demo [here](https://codesandbox.io/s/sveltejskit-framework7-playground-forked-14mno) but it uses some older versions and I can't make it work on my environment with latest versions of svelte and sveltekit.

**Comment by D-Marc1** [comment]
This really needs to be a built-in F7 feature. I think @nolimits4web is making a mistake not having native SSR support for F7.

However, KonstaUI has SvelteKit support and is made by the same author as F7. The only downside is that it's missing quite a few UI components that exist in F7.

**Comment by MarcGodard** [comment]
I agree completely. With PWA builder, you can really build a nice app with Framework 7 + svelteKit. Plus all the awesome routing ability, you can really create some powerful stuff.

**Comment by Andre7th** [comment]
please make a template with sveltekit and f7

**Comment by nolimits4web** [comment]
This issue was incorrectly closed on 2026-05-19 by a compromised credential (Shai-Hulud npm supply-chain worm). Reopening — apologies for the noise.
